crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,12 +1276,20 @@ def poweron
 
   def powercycle
     set_state("reboot")
-    bmc_cmd("power cycle")
+    if @node[:platform] == "windows"
+      net_rpc_cmd(:power_cycle)
```

Comment:
> It can be executed also without -f, but to force dangling applications to close is safer.

I think that's partly why we have the two different methods. A reboot is supposed to be clean, so without `-f`.